src/components/frame/v5/pages/UserCryptoToFiatPage/partials/LiquidationAddress/LiquidationAddress.tsx

```diff
@@ -0,0 +1,159 @@
+import { Cardholder, Eye } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React, { useEffect, useState } from 'react';
+import { defineMessages } from 'react-intl';
+
+import LoadingSkeleton from '~common/LoadingSkeleton/index.ts';
+import { useAppContext } from '~context/AppContext/AppContext.ts';
+import { useGetUserLiquidationAddressesLazyQuery } from '~gql';
+import useCopyToClipboard from '~hooks/useCopyToClipboard.ts';
+import { getChainId } from '~redux/sagas/utils/index.ts';
+import { formatText } from '~utils/intl.ts';
+import Button from '~v5/shared/Button/index.ts';
+
+import { useCryptoToFiatContext } from '../../context/CryptoToFiatContext.ts';
+
+import { MOCK_LIQUIDATION_ADDRESS_CHAIN_ID } from './mocks.ts';
+import { BlockExplorerButton } from './partials/BlockExplorerButton.tsx';
+
+const displayName = 'v5.pages.UserCryptoToFiatPage.partials.LiquidationAddress';
+
+const chainIdWithMockFallback = Number(
```

Comment:
Could you add a TODO here to change this once the hardcoded chain ID is removed from the bridgeXYZMutation lambda?

Reply:
All right I'll copy over the todo comment I have in the other file 😂